firestore-send-email/integration-test/firestore.rules

```diff
@@ -0,0 +1,198 @@
+rules_version = '2';
```

Comment:
Looks like these were exported from the extensions-testing project by firebase init. They can probably be replaced with default rules files for the purposes of this test.